### PR TITLE
Implement #3045, mark old methods as deprecated

### PR DESCRIPTION
**Pull request by Sylvyrfysh**
Makes method names in wallet_rpc consistent. Old methods are left in to be removed later.

### CODE REVIEW

**Comment by moneromooo-monero** [comment]
If you want to have "get_balance" instead of "getbalance", I don't think you need another function, just another json name pointing to the same.

**Comment by Sylvyrfysh** [comment]
Implemented as suggested, everything squashed.

File: src/wallet/wallet_rpc_server.h [hunk 1]
```diff
@@ -78,7 +82,6 @@ namespace tools
         MAP_JON_RPC_WE("tag_accounts",       on_tag_accounts,       wallet_rpc::COMMAND_RPC_TAG_ACCOUNTS)
         MAP_JON_RPC_WE("untag_accounts",     on_untag_accounts,     wallet_rpc::COMMAND_RPC_UNTAG_ACCOUNTS)
         MAP_JON_RPC_WE("set_account_tag_description", on_set_account_tag_description, wallet_rpc::COMMAND_RPC_SET_ACCOUNT_TAG_DESCRIPTION)
-        MAP_JON_RPC_WE("getheight",          on_getheight,          wallet_rpc::COMMAND_RPC_GET_HEIGHT)
```
Comment (stoffu) [inline src/wallet/wallet_rpc_server.h]: I don't see a reason to move this line. Instead, the overloaded version could be added just above/below this line.

Comment (Sylvyrfysh) [reply src/wallet/wallet_rpc_server.h]: Changed

**Comment by onodera-punpun** [comment]
I just noticed that the [daemon RPC](https://getmonero.org/resources/developer-guides/daemon-rpc.html) also has a few of these non snakecase methods, maybe they could be added to this PR as well.

**Comment by Sylvyrfysh** [comment]
@moneromooo-monero @stoffu [Daemon RPC docs](https://getmonero.org/resources/developer-guides/daemon-rpc.html) (see setbans) and code `MAP_JON_RPC_WE_IF("set_bans", ,,,)` are not consistent. This should be a website fix, correct?

**Comment by moneromooo-monero** [comment]
tbh, if this is going to become a full scale makeup saloon, I'm not too keen on it. A few changes are fine, but if it's going to be dozens, I'd rather just leave it at that, it's not like it's breaking anything.

**Pull request merged**